scripts/prettier/index.js

```diff
@@ -99,28 +95,46 @@ Object.keys(config).forEach(key => {
     return;
   }
 
-  const args = Object.keys(defaultOptions).map(
-    k => `--${k}=${(options && options[k]) || defaultOptions[k]}`
-  );
-  args.push(`--${shouldWrite ? 'write' : 'l'}`);
-
-  try {
-    exec(prettierCmd, [...args, ...files]).trim();
-  } catch (e) {
-    if (!shouldWrite) {
-      console.log(
-        '\n' +
-          chalk.red(
-            `  This project uses prettier to format all JavaScript code.\n`
-          ) +
-          chalk.dim(`    Please run `) +
-          chalk.reset('yarn prettier-all') +
-          chalk.dim(` and add changes to files listed below to your commit:`) +
-          `\n\n` +
-          e.stdout
-      );
-      process.exit(1);
+  const args = Object.keys(defaultOptions).reduce((acc, _key) => {
+    const camelize = /[\-\:]([a-z])/g;
+    const capitalize = token => token[1].toUpperCase();
+    const prettierKey = _key.replace(camelize, capitalize);
```

Comment:
This is unnecessary. Just convert the options to use camel case form instead where they are defined.